docs/source/models/other_models/alzheimers_population/index.rst

```diff
@@ -30,4 +30,230 @@
+
+.. note::
+
+  Another way to write :eq:`initial_pop_eq` is
+
+  .. math::
+
+    X_{g,t_0} = X_{t_0}
+    \cdot \frac{\text{# of real people in subgroup $g$ with Alzheimer's}}
+      {\text{# of real people in whole population with Alzheimer's}}.
+
+  Thus, we could compute :math:`X_{g,t_0}` using prevalence counts from
+  GBD instead of prevalence rates.
+
+  To verify that :eq:`initial_pop_eq` gives us the correct total number of
+  initial simulants, note that
+
+  .. math::
+
+    \begin{align*}
+    \sum_g X_{g,t_0}
+    = \sum_g X_{t_0}
+      \cdot \frac{p_{g,t_0} \cdot \hat X^\text{real}_{g,t_0}}
+      {p_\text{Alz} \cdot \hat X^\text{real}_{t_0}}
+    &= X_{t_0} \cdot \sum_g
+      \frac{X^\text{real}_{g,t_0}}{X^\text{real}_{t_0}} \\
+    &= X_{t_0} \cdot
+      \frac{\sum_g X^\text{real}_{g,t_0}}{X^\text{real}_{t_0}}
+    = X_{t_0} \cdot
+      \frac{X^\text{real}_{t_0}}{X^\text{real}_{t_0}}
+    = X_{t_0}.
+    \end{align*}
+
+Adding New Simulants
+++++++++++++++++++++
+
+Let :math:`N_{g,t}` denote the number of new simulants in demographic
+group :math:`g` that we want to add to the simulation at time :math:`t`.
+We will assume that :math:`N_{g,t}` is a Poisson random variable with
+mean :math:`\lambda_{g,t} \cdot \Delta t \cdot 1_{\{\text{simulation
+step times}\}}(t)`, where :math:`\lambda_{g,t}` is the entrance rate of
+new simulants (measured in count of simulants per unit time) at time
+:math:`t`, :math:`\Delta t` is the length of a simulation time step, and
+:math:`1_A` is the indicator function of the set :math:`A` (the
+indicator function zeros out the entrance rate at times when the
+simulation is not taking a step). Our goal is to determine the entrance
+rate :math:`\lambda_{g,t}` for each :math:`g` and :math:`t`.
+
+Let :math:`A_g(t)` be the cumulative number of incident cases of AD by
+time :math:`t` in demographic group :math:`g` in the real population.
+Since our simulation is scaled down by a factor of :math:`S`, the rate
+at which we want to add simulants is
+
+.. math::
+
+  \lambda_{g,t} = S \cdot A_g'(t).
+
+We rewrite this in terms of quantities that we can estimate from the
+available data:
+
+.. math::
+  :label: AD_entrance_rate_eq
+
+  \lambda_{g,t}
+  = S \cdot A_g'(t)
+  = S \cdot \frac{A_g'(t)}{\hat X^\text{real}_{g,t}}
+    \cdot \hat X^\text{real}_{g,t}
+  = S \cdot i_{g,t} \cdot \hat X^\text{real}_{g,t},
+
+where :math:`i_{g,t} = A_g'(t) /\hat X^\text{real}_{g,t}` is the **total
+population incidence hazard** of AD in demographic group :math:`g` at
+time :math:`t`. We know the model scale :math:`S` from
+:eq:`model_scale_eq` above, and we can estimate the quantities
+:math:`i_{g,t}` and :math:`X^\text{real}_{g,t}` from GBD as follows.
+
+Let :math:`y(t)` denote the year in which time :math:`t` occurs. If we
+assume that the hazard :math:`i_{g,t}` is constant throughout the year
+:math:`y(t)`, then it is equal to its person-time-average over the year,
+which is the **total population incidence rate**:
```

Comment:
Link to vivarium_research page on total population incidence rate? I thought we had a page on this, but I can't find it now... https://vivarium-research.readthedocs.io/en/stable/model_design/general_reference_material/measures_of_risk/index.html#incidence

Reply:
It's here: https://vivarium-research.readthedocs.io/en/latest/model_design/vivarium_model_components/causes/index.html#incidence-rates

It's a bit confusing that we have two different pages that talk about incidence rates...